src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -28,7 +27,8 @@ public String index() {
      * @return disorders page
      */
     @GetMapping("/disorders")
-    public ResponseEntity fetchAllDisorders() {
+    public ResponseEntity fetchAllDisorders()
```

Comment:
cleaned up method signature

Reply:
Curlies are programmer's preference.  What I recommend is defining a format for all group members, and then using the IDE's Control-Alt-S shortcut to apply it uniformally.

Reply:
Having the brackets in a separate line is a C# convention that Visual Studio enforces. In java, as part of conventions, the  beginning bracket is on the same line as the method name and args (if any)

Reply:
We're opting to use java's default convetions